gpt4all-chat/CMakeLists.txt

```diff
@@ -68,9 +68,21 @@ if(${CMAKE_SYSTEM_NAME} MATCHES Darwin)
   set(METAL_SHADER_FILE ../gpt4all-backend/llama.cpp-mainline/ggml-metal.metal)
 endif()
 
-set(APP_ICON_FILE "${CMAKE_CURRENT_SOURCE_DIR}/icons/favicon.icns")
-set_source_files_properties(${APP_ICON_FILE} PROPERTIES
-    MACOSX_PACKAGE_LOCATION "Resources")
+set(APP_ICON_RESOURCE)
+if (WIN32)
+    set(APP_ICON_RESOURCE "${CMAKE_CURRENT_SOURCE_DIR}/resources/gpt4all.rc")
+elseif (APPLE)
+    # The MACOSX_BUNDLE_ICON_FILE variable is added to the Info.plist
```

Comment:
You mentioned that you tested on windows, but how about mac? This includes changes to mac so have you tested and verified it works there as well?

Reply:
I do remember testing it on macOS at one point and the app still had an icon. I'll test it again to be sure.

Reply:
Yes, I am still seeing all of the expected icons on macOS.